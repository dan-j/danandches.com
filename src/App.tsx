import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { Route } from 'react-router';
import Header from './components/common/Header';
import Home from './components/Home';

export default class App extends React.Component<{}, {}> {

    render() {
        return (
            <BrowserRouter>
                <div>
                    <Header />
                    <Route path="/">
                        <Home />
                    </Route>
                </div>
            </BrowserRouter>
        );
    }
}
