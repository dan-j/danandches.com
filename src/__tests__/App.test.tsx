import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json/';

import App from '../App';

describe('<App />', () => {
    it('Renders a default greeting', () => {
        const wrapper = shallow(<App />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});

