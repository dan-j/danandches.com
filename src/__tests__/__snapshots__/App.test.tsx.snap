// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Renders a default greeting 1`] = `
<BrowserRouter>
  <div>
    <Header />
    <Route
      path="/"
    >
      <Home />
    </Route>
  </div>
</BrowserRouter>
`;
